static/app/components/performance/waterfall/rowDivider.tsx

```diff
@@ -39,3 +47,26 @@ export const DividerLineGhostContainer = styled('div')`
   width: 100%;
   height: 100%;
 `;
+
+const BadgeBorder = styled('div')`
+  position: absolute;
+  margin: ${space(0.25)};
+  left: -11.5px;
```

Comment:
Decimal pixel value, hmm 🤔

Reply:
this decimal actually doesnt do anything, removed in 2aa562e